src/tutorials/lua_basics_lightbulb.md

```diff
@@ -299,54 +319,59 @@ This function will define what happens when the player interacts with the trigge
 
 Eventually we want the switch to flip up and down when the player interacts with it, turning the light on and off. For now we'll just place our rotate statement inside it, which is just the switch turning down.
 
-Cut and paste the `RotateTo` statement into our `onInteraction` function.
+Cut and paste the `RotateTo` statement into our `OnTriggerInteracted` function.
 
 It should now look like this:
 
 ```lua
-local switch = script.parent
-local switchTrigger = switch:FindChildByName("Trigger")
-
-local onRotation = Rotation.New(0, -60, -0)
-local offRotation = Rotation.New(-180, -60, 180)
+local Switch = script:GetCustomProperty("Switch"):WaitForObject()
+local Light = script:GetCustomProperty("Light"):WaitForObject()
+local Trigger = script:GetCustomProperty("Trigger"):WaitForObject()
 
+local ROTATION_ON = Rotation.New(0, -60, 0)
+local ROTATION_OFF = Rotation.New(0, -130, 0)
+local TIME_ROTATE = 2
 
-local function OnSwitchInteraction()
-    switch:RotateTo(onRotation, 2, true)
+function OnTriggerInteracted()
+    Switch:RotateTo(ROTATION_ON, TIME_ROTATE, true)
 end
 ```
 
 ### Connect the Function to an Event
 
-Lastly, you will need an event statement that tells the script to do the `OnSwitchInteraction` function when the player interacts with the trigger. At the end of your script type:
+Lastly, you will need an event statement that tells the script to do the `OnTriggerInteracted` function when the player interacts with the trigger. At the end of your script type:
 
 ```lua
-switchTrigger.interactedEvent:Connect(OnSwitchInteraction)
+Trigger.interactedEvent:Connect(OnTriggerInteracted)
 ```
 
-- `switchTrigger` is the variable name of this trigger.
-- `interactedEvent:Connect()` tells the script every time the player interacts with trigger to execute the function passed to the `Connect()` function.
-- `OnSwitchInteraction` is the name of the function we are connecting.
+- `Trigger` is the variable name of this trigger.
+- `interactedEvent:Connect(function)` tells the script every time the player interacts with trigger to execute the function passed to the `Connect()` function.
+- `OnTriggerInteracted` is the name of the function we are connecting.
 
 Without this statement the script will never call the `OnSwitchInteraction` function.
 
+!!! note
+    It is convention to name the callback of an event (the function that is called when the event is fired) to be `On` + **Event Name** in PascalCase. So, for this example, we would use `OnInteracted` but decided to use `OnTriggerInteracted` to provide more clarity.
```

Comment:
Take out "It is convention", and use the explanation of callback in place of the word. 

Also is this really in our style guide? I would either skip the explanation, or focus on why the choice is clear.

Also, CamelCase or UpperCamelCase. Ain't nobody need to worry about Pascal.